### PR TITLE
LDAP: users without org mappings are marked as disabled

### PR DESCRIPTION
**Pull request by xlson**
**What this PR does / why we need it**:

This PR corrects the mapping of `extUser` so that it knows that a user that does not have org mappings (where org mappings are expected by the ldap config) is disabled.

This will make it so that LDAP Debug view will correctly show disabled for users that don't have any valid mappings and also make it so that the LDAP background sync disables users that no longer have valid org mappings.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/services/ldap/ldap.go [hunk 1]
```diff
@@ -427,6 +427,12 @@ func (server *Server) buildGrafanaUser(user *ldap.Entry) (*models.ExternalUserIn
 		}
 	}
 
+	// If there are groups org mappings configured but no matching mappings
```
Comment (aknuds1) [inline pkg/services/ldap/ldap.go:430]: I think "group org mappings" is correct?
```suggestion
	// If there are group org mappings configured, but no matching mappings,
```

**Review by aknuds1 — APPROVED** [review]
LGTM with a language suggestion.

Could any test be written for this?

**Review by alexanderzobnin — APPROVED** [review]
LGTM!

**Comment by xlson** [comment]
> LGTM with a language suggestion.
> 
> Could any test be written for this?

Good point. I've added tests to cover this behaviour.

**Pull request merged**